examples/kubernetes/nfs-provisioner/README.md

```diff
@@ -0,0 +1,13 @@
+# Set up a NFS Server on a Kubernetes cluster
+
+### 1. Create a NFS Server Deployment
+
+```bash
+kubectl create -f https://raw.githubusercontent.com/kubernetes-csi/nfs-driver-smb/master/examples/kubernetes/nfs-provisioner/nfs-server.yaml
+```
+
+### 2. Create a PersistentVolume
```

Comment:
PV is not part of this nfs-provisioner config, may move this example to other place

Reply:
Sure, I have removed it for now. Added example for getting public IP using LoadBalancer instead

Reply:
pls use `ClusterIP` by default, `LoadBalancer` requires cloud provider, let's not depend on cloud provider in this project

Reply:
Yes the `nfs-server.yaml` uses ClusterIP. `nfs-server-lb.yaml` uses LB. Shall I remove that example or keep it optional?

Reply:
keep it as optional is fine

Reply:
Sure, the updated readme mentions nfs-server.yaml as default. I could make changes there if required, thanks